### DISCUSSION TITLE
Is it possible to use more than 2 types of regex for a single type of card

### DISCUSSION THREAD

**Discussion by Bestboigill** [ObsidianToAnki/Obsidian_to_Anki#626] [Q&A] [answered]
for example using inline regex and tag regex together , offcource it would be possible to use a complex regex but it wouldnt be ideal

**Comment by envico801** [comment, marked as answer]
Hi @Bestboigill, yes, it can be done in 2 ways:

**Option A**:

You can clone the card type to which you want to assign a regex inside anki, then you would have to regenerate the table of note types inside the plugin in obsidian (so that it is updated) and there assign the different regex to each note type (which would be the same technically).

---

**Option B**:

As you mention, you could modify the regex to combine both regex expressions into one, for example:

```text
Regex: /(Question selector 1 | Question selector 2) (Answer selector 1 | Answer selector 2)/g
```
